docs/sources/flow/getting-started/opentelemetry-to-lgtm-stack.md

```diff
@@ -0,0 +1,318 @@
+			username = 5252
+			password = env("GRAFANA_CLOUD_API_KEY")
+		}
+	}
+}
+```
+
+[Grafana Loki]: https://grafana.com/oss/loki/
+
+## Grafana Tempo
+
+[Grafana Tempo][] is an open-source, easy-to-use, scalable distributed tracing backend. Tempo can ingest OTLP directly, and you can use the OTLP exporter to send the traces to Tempo.
+
+```river
+otelcol.exporter.otlp "default" {
+  client {
+    endpoint = "tempo-server:4317"
+  }
+}
+```
+
+To use Tempo with basic-auth, which is required with GrafanaCloud Tempo, you must use the [otelcol.auth.basic][] component. You can get the Tempo config from the Tempo **Details** page in the [GrafanaCloud Portal][]:
+
+![](../../../assets/getting-started/tempo-config.png)
+
+```river
+otelcol.exporter.otlp "grafana_cloud_tempo" {
+	client {
+		endpoint = "tempo-us-central1.grafana.net:443"
+		auth     = otelcol.auth.basic.grafana_cloud_tempo.handler
+	}
+}
+
+otelcol.auth.basic "grafana_cloud_tempo" {
+	username = 4094
+	password = env("GRAFANA_CLOUD_API_KEY")
+}
+```
+
+[Grafana Tempo]: https://grafana.com/oss/tempo/
+[GrafanaCloud Portal]: https://grafana.com/docs/grafana-cloud/account-management/cloud-portal/#your-grafana-cloud-stack 
+
+## Grafana Mimir / Prometheus Remote Write
+
+[Prometheus Remote Write] is a popular metrics transmission protocol supported by most metrics system, including [Grafana Mimir] and GrafanaCloud. To send from OTLP to Prometheus, we do a passthrough from the [otelcol.exporter.prometheus][] to the [prometheus.remote_write][] component. The Agent’s Prometheus remote write is a more battle-tested implementation with a Write Ahead Log for resiliency.
+
+```river
+otelcol.exporter.prometheus "default" {
+	forward_to = [prometheus.remote_write.default.receiver]
+}
+
+prometheus.remote_write "default" {
+	endpoint {
+		url = "http://prometheus:9090/api/v1/write"
+	}
+}
+```
+
+To use Prometheus with basic-auth, which is required with GrafanaCloud Prometheus, you have to configure the [prometheus.remote_write][] component. You can get the Prometheus config from the Prometheus **Details** page in the [GrafanaCloud Portal][]:
+
+![](../../../assets/getting-started/prometheus-config.png)
+
+```river
+otelcol.exporter.prometheus "grafana_cloud_prometheus" {
+        forward_to = [prometheus.remote_write.grafana_cloud_prometheus.receiver]
+    }
+
+prometheus.remote_write "grafana_cloud_prometheus" {
+    endpoint {
+        url = "https://prometheus-us-central1.grafana.net/api/prom/push"
+
+        basic_auth {
+            username = 12690
+            password = env("GRAFANA_CLOUD_API_KEY")
+        }
+    }
+}
+```
+
+[Prometheus Remote Write]: https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage
+[Grafana Mimir]: https://grafana.com/oss/mimir/
+
+## Putting it all together
+
+Instead of referencing `otelcol.exporter.otlp.default.input` in the output of `otelcol.processor.batch`, we need to reference the three exporters we set up. The final config becomes:
```

Comment:
```suggestion
Instead of referencing `otelcol.exporter.otlp.default.input` in the output of `otelcol.processor.batch`, we need to reference the three exporters we set up. The final configuration becomes:
```